e2e/stack_test.go

```diff
@@ -158,7 +162,22 @@ func TestE2E(t *testing.T) {
 	client, err := bftclient.New(monomerCometURL.String(), monomerCometURL.String())
 	require.NoError(t, err, "create Comet client")
 
-	txBytes := testapp.ToTx(t, "userTxKey", "userTxValue")
+	app := testapp.NewTest(t, chainID.String())
```

Comment:
This is a common pattern across every test that needs to sign transactions with testapp.ToTx. We need to call InitChain before we fetch the application context, and then we get our pre-funded test account with app.TestAccount.

It'd probably be good to encapsulate this setup logic in a helper function (perhaps inside of testapp.NewTest itself), so we can avoid repeating it across each test.